source/algorithms.tex

```diff
@@ -1126,39 +1126,39 @@
 or
 \tcode{Input\-Iterator2},
 the template argument shall satisfy the
-requirements of an input iterator\iref{input.iterators}.
+\textit{Cpp98InputIterator} requirements\iref{input.iterators}.
 \item
 If an algorithm's template parameter is named
 \tcode{OutputIterator},
 \tcode{OutputIterator1},
 or
 \tcode{Output\-Iterator2},
-the template argument shall satisfy the requirements
-of an output iterator\iref{output.iterators}.
+the template argument shall satisfy the
+\textit{Cpp98OutputIterator} requirements\iref{output.iterators}.
```

Comment:
I'm wondering whether we can find a slightly less awkward spelling than "Cpp98Blah" for these old-style concepts. Maybe even something innovative such as OutputIterator<sub>98</sub> would work.

Reply:
I'm certainly not married to "Cpp98Blah". It's better than LEWG's suggestion "STL1Blah", but that wasn't a high bar to meet. Now that the styling is macro-ized, we can change it with a one-liner.

Reply:
_Cpp98MoveConstructible_ certainly feels anachronistic.

Reply:
> Cpp98MoveConstructible certainly feels anachronistic.

We specifically pointed out that example in both LEWG discussion of the Ranges proposal and LWG discussion of this proposal, and both subgroups found the confusion level acceptable.